torch/csrc/jit/passes/quantization.cpp

```diff
@@ -898,6 +921,8 @@ graph(%linear, %a_dequant, %w, %b, %w_scale, %w_zero_point, %w_dtype):
   SubgraphRewriter rewriter;
   rewriter.RegisterRewritePattern(linear_with_quant, linear_with_quant_prepack);
   rewriter.runOnGraph(graph, filter);
+
+  insertPrepackUnpackForConv2d(graph);
```

Comment:
Code above should be factored out into a separate function (or `insertPrepackUnpackForConv2d` should be inlined here). Otherwise, it's inconsistent on the level of abstraction/details.

Reply:
yeah, planning to refactor the above into a different function as well, will do in a separate PR